ephios/core/templates/core/home.html

```diff
@@ -52,7 +52,10 @@ <h5 class="card-title mb-0">{% trans "Events with requested participations" %}</
                     {% for shift in shifts %}
                         <li class="list-group-item flex-column">
                             <div class="d-flex w-100 justify-content-between align-items-center">
-                                <span>{{ shift.event.title }} ({{ shift.get_start_end_time_display }})</span>
+                                <span>
+                                    <span class="badge bg-warning">{{ shift.request_count }} {% translate "requests" %}</span>
```

Comment:
What do you think about this?

Reply:
I think it's too intrusive. If there are multiple events in the list there is a lot of yellow on the screen, and there is no reason to highlight every single entry because they all need similar attention regardless of the count of requests.
I think i'd prefer finding a small indicator (such as the number in the round label that you proposed before) that we can use here and on the event list to have a common look across the views

Reply:
will remove